src/main/resources/assets/structurize/lang/en_us.json

```diff
@@ -350,10 +350,10 @@
   "block.structurize.framed_jungle_acacia_timber_frame": "Framed Jungle Acacia Timber Frame",
   "block.structurize.gated_oak_jungle_timber_frame": "Gate Framed Oak Jungle Timber Frame",
   "block.structurize.side_framed_oak_jungle_timber_frame": "Side Framed Oak Jungle Timber Frame",
-  "block.structurize.jungle_blockpaperwall": "Jungle Paperwall",
+  "block.structurize.jungle_blockpaperwall": "Jungle Paper Wall",
   "block.structurize.plain_cactus_jungle_timber_frame": "Plain Cactus Jungle Timber Frame",
   "block.structurize.gated_cactus_oak_timber_frame": "Gate Framed Cactus Oak Timber Frame",
-  "structurize.command.ls.create.done": "Created session for player: %s",
+  "structurize.command.ls.create.done": "Created session for player %s.",
```

Comment:
Then do \"%s\" so the nickname is separated from text

Reply:
What do you mean? Does "%s" put it on a new line or something?

Reply:
nah, just to make sure nickname has somewhat borders in the text
like `Created session for player "Nightenom".`
thus the translation would be `Created session for player "%s".`